docs/source/our_stack.rst

```diff
@@ -10,31 +10,104 @@ The most up-to-date reference on our codebase is our code itself.
     To see when any documentation page was last updated, click "Edit on GitHub" in
     the top right, then see the timestamp on the last commit.
 
-...
-TODO(Kevin): stub out the rest
-
-Overview
+Gameplay
 --------
+Gameplay is where our strategy is produced. It takes in the current state and
+assigns all robots a task based on some coordination. Our
+system utilizes a play selector that chooses a play based on world state. After
+that play assigns roles, each role selects a tactic based on their progress, and
+tactic select an atomic skill to perform. The gameplay loop is set to tick at
+60Hz. The code block below describes this process. Each time the timer
+ends, the callback method is entered:
 
-ROS2
-----
+.. code-block:: python
 
-Vision
-------
+        timer_period = 1 / 60  # seconds
+        self.timer = self.create_timer(timer_period, self.gameplay_tick)
 
-Gameplay
---------
+Motion Control
+--------------
+A lot of the motion control is done on robot by the firmware, but part of the
+motion control is handled by software. See the ``control`` directory,
+``motion_control.cpp`` uses PID to correct robot position. The PID
+controller implementation is not in software, but it is in the
+`fshare repository <https://github.com/RoboJackets/robocup-fshare>`_.
+Correction of robot velocity is done by robot firmware (the code that runs on
+each robot's microcontroller) in the separate firmware repository.
 
 Motion Planning
 ---------------
+Motion planning is about generating a path from a robot's current position and
+orientation (pose) to a goal supplied by gameplay. Starting at the lowest
+level, we use `RRT <https://www.youtube.com/watch?v=Ob3BIJkQJEw>`, or Rapidly exploring Random Trees, for path generation. When the straight line path to a goal fails, we utilize the path generated by RRT (if there are no
```

Comment:
```suggestion
level, we use `RRT <https://www.youtube.com/watch?v=Ob3BIJkQJEw>`_, or Rapidly exploring Random Trees, for path generation. When the straight line path to a goal fails, we utilize the path generated by RRT (if there are no
```
https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages